series/praha-pyvo/events/2018-03-21.yaml

```diff
@@ -0,0 +1,17 @@
+city: praha
+start: 2018-02-21 19:00:00
```

Comment:
Tady chceš `2018-03-21`

Reply:
ciwe.. jsem to asi editoval v jinym souboru nebo co :D